src/constants.rs

```diff
@@ -0,0 +1,55 @@
+//
+// constants.rs
```

Comment:
In general, I don't think a bunch of constants warrant their own module. These should probably stay in `lib.rs`.

But, for that matter, all of these constants and more are now available from the `libc` crate. We should probably import and re-export all of the ones related to CAN, like:

```
// lib.rs

pub use libc::{
    AF_CAN,
    PF_CAN,
    ...
};
```